src/client/manager/provider.js

```diff
@@ -4,6 +4,7 @@ import React from 'react';
 import { Provider } from '@kadira/storybook-ui';
 import addons from '@kadira/storybook-addons';
 import createChannel from '@kadira/storybook-channel-pagebus';
+import createDatabase from '@kadira/storybook-database-local';
```

Comment:
We need this on RN storybook also. I'll move the middle ware tomorrow. Please hold this PR until then.